samza-core/src/test/java/org/apache/samza/startpoint/TestStartpointSerde.java

```diff
@@ -18,45 +18,47 @@
  */
 package org.apache.samza.startpoint;
 
+import java.io.IOException;
+import org.codehaus.jackson.map.ObjectMapper;
 import org.junit.Assert;
 import org.junit.Test;
 
 public class TestStartpointSerde {
```

Comment:
Maybe combine this with the other startpoint object mapper tests, since there isn't an actual `StartpointSerde` class anymore.